internal/govmomi/host_internal_config_manager.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
```

Comment:
Go documentation says it is possible ```any package in or under a directory named internal may be imported by packages rooted in the same subtree.``` but I not sure that is what we want as that naming dictates that only pkv/vsphere can import that directory.